tensorboard/backend/json_util.py

```diff
@@ -68,5 +70,7 @@ def Cleanse(obj, encoding="utf-8"):
         return collections.OrderedDict(
             (Cleanse(k, encoding), Cleanse(v, encoding)) for k, v in obj.items()
         )
+    elif dataclasses.is_dataclass(obj):
+        return [Cleanse(i, encoding) for i in dataclasses.asdict(obj).values()]
```

Comment:
Originally I tried turning it just into a mapping then the distributions plugin won't load (
although the other common plugins seem fine): 
![image](https://user-images.githubusercontent.com/15273931/198310689-a8a9f947-0f07-4371-9ba1-a88c85ae6708.png)

Changed to use `astuple()` for cleaner code, thanks for the suggestion!

Reply:
Ah, okay. I think that's just because the distributions plugin calls `compress_histogram()` which returns what was previously a namedtuple. The relevant line is here: https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/plugins/distribution/distributions_plugin.py;l=93;drc=b306651ab5baa6d362326464a703463b4683b265  That value later gets returned in a JSON HTTP response.

So rather than changing the generic json_util.py to handle any dataclass as a tuple, I think it's a bit more targeted to just do an extra conversion at that callsite to convert the CompressedHistogramValue dataclass back into a tuple (e.g. call astuple() there instead, explicitly).

Reply:
Thanks for the suggestion! Added a `as_tuple` class method in `CompressedHistogramValue` and reverted the changes in `json_util`.